<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DifficultyDialog</class>
 <widget class="QDialog" name="DifficultyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>192</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="Label" name="cTitle">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>385</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string>Title</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Label" name="cDifficulty">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>54</y>
     <width>284</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Difficulty</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Button" name="cUp">
   <property name="geometry">
    <rect>
     <x>288</x>
     <y>54</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Button" name="cDown">
   <property name="geometry">
    <rect>
     <x>312</x>
     <y>54</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Label" name="cGodEffects">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>94</y>
     <width>284</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>God Effects</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Button" name="cRight">
   <property name="geometry">
    <rect>
     <x>288</x>
     <y>94</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Label" name="cExit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>164</y>
     <width>384</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Button</class>
   <extends>QPushButton</extends>
   <header>widget/button.h</header>
  </customwidget>
  <customwidget>
   <class>Label</class>
   <extends>QLabel</extends>
   <header>widget/label.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
